app/core/auth.coffee

```diff
@@ -22,14 +22,13 @@ init = ->
     me.patch()
   setTestGroupNumberUS()
   preferredLanguage = getQueryVariable('preferredLanguage')
-  if me and features.codePlay and preferredLanguage
+  if me and preferredLanguage
```

Comment:
I don't see why we wouldn't set the preferred language here, but perhaps this is wrong?

Reply:
This is fine, we don't set the `preferredLanguage` query variable as far as I know now, but if we did, it'd be handy to have it work.